bugbug/bug_features.py

```diff
@@ -209,6 +210,45 @@ def __call__(self, bug, **kwargs):
         return None
 
 
+class delta_nightly_request_merge(single_bug_feature):
+    name = "Time delta between landing of the patch in Nightly and uplift request"
+
+    def __call__(self, bug, **kwargs):
+        for history in bug["history"]:
+            for change in history["changes"]:
+                if change["added"].startswith("approval-mozilla"):
```

Comment:
This will match uplift requests and uplift approvals as well.
```suggestion
                if change["added"].startswith("approval-mozilla") and change["added"].endswith("?"):
```

Ending with  `?` means the uplift was requested; when the uplift gets approved, it gets changed to `+`. See the example below:
```json
{
    "when": "2019-09-11T19:45:37Z",
    "changes": [
        {
        "attachment_id": 9088191,
        "removed": "approval-mozilla-beta?",
        "added": "approval-mozilla-beta+",
        "field_name": "flagtypes.name"
        }
    ],
    "who": "lhenry@mozilla.com"
},
```

Reply:
accepted changes